lib/transports/rest2.js

```diff
@@ -250,6 +252,52 @@ class RESTv2 {
     return new ModelClass(data)
   }
 
+  /**
+   * @param {Array[]} data order matters
+   * @return {Arr} merged arr of currencies and names sorted with no pairs repeated
+   * @private
+   */
+
+  _mergeCurrenciesListSymLabel (data) {
+    if (!Array.isArray(data) && data.length !== 3) {
+      return data
+    }
+
+    const listedCurr = data[0]
+    const mapedCurrSym = data[1]
+    const mapedCurrLabel = data[2]
+    const listedCurrAsArr = []
+
+    listedCurr.forEach((c) => {
+      listedCurrAsArr.push([ c, c ])
+    })
+
+    const allCurr = [
+      ...mapedCurrLabel,
+      ...mapedCurrSym,
+      ...listedCurrAsArr
+    ]
+
+    const indexNotInArr = (arr, c) => {
+      return arr.every((el) => {
+        return el[0] !== c[0]
+      })
+    }
+
+    const allNotRepeatedCurr = []
+    allCurr.forEach((c) => {
+      if (indexNotInArr(allNotRepeatedCurr, c)) {
```

Comment:
Pls use `_includes` https://lodash.com/docs/4.17.10#includes

You can also build a map by currency, and then use `Object.keys()` to ensure there are no duplicates, it may be faster

Reply:
lodash includes does not work for this as it what is it asking if the first item of an array is included on the first item of the arrays, example: